Userland/Games/Solitaire/Game.h

```diff
@@ -182,7 +182,6 @@ class Game final : public Cards::CardGame {
     void auto_move_eligible_cards_to_foundations();
     void start_timer_if_necessary();
     void start_game_over_animation();
-    void stop_game_over_animation();
```

Comment:
Now we have a method to start the animation but not one to stop it, which doesn't seem great?

Reply:
:thonk: Good point, I'll take a look later, I might not have needed to remove it.